packages/code-du-travail-frontend/src/common/Answer.js

```diff
@@ -45,97 +47,72 @@ function Answer({
-        Array.from(node.querySelectorAll("[data-tooltip-slug]")).map(node => ({
-          node,
-          term: node.getAttribute("data-tooltip-term"),
-          definition:
-            glossaryBySlug[node.getAttribute("data-tooltip-slug")].definition
-        }))
-      );
-    }, []);
-    setPortalComponents(
-      nodes.map(({ node, term, definition }, i) => {
-        return (
-          <Portal key={`item-${i}`} node={node}>
-            <DefinitonTerm term={term} definition={definition} />
-          </Portal>
-        );
-      })
-    );
-    return function cleanEffect() {};
-  }, [children, html]);
   return (
     <>
       <Head>
         <title>{title}</title>
       </Head>
       <ThemeBreadcrumbs breadcrumbs={breadcrumbs} />
       <BackToResultsLink query={router.query} />
-      {!html && !children && <BigError>{emptyMessage}</BigError>}
-      {(html || children) && (
-        <Article
-          wide={wide}
-          title={title}
-          icon={icon}
-          date={date}
-          sourceType={sourceType}
-        >
-          {intro && <IntroWrapper variant="dark">{intro}</IntroWrapper>}
-          {html && <Html>{html}</Html>}
-          {children}
-          {portalComponents}
-          {footer && <Footer>{footer}</Footer>}
-        </Article>
-      )}
-      {additionalContent}
-      <Feedback
-        query={router.query.q}
-        sourceType={sourceType}
-        sourceFilter={router.query.source}
-        url={router.asPath}
-        title={title}
-      />
+      <StyledWrapper>
+        <StyledContent isFull={linkedResults.matches.length === 0}>
+          {!html && !children && <BigError>{emptyMessage}</BigError>}
+          {(html || children) && (
+            <Article
+              wide={wide}
+              title={title}
+              date={date}
+              sourceType={sourceType}
+            >
+              {intro && <IntroWrapper variant="dark">{intro}</IntroWrapper>}
+              {html && <Html>{html}</Html>}
+              {children}
+              {portalComponents}
+              {footer && <Footer>{footer}</Footer>}
+            </Article>
+          )}
+          {additionalContent}
+          <Feedback
+            query={router.query.q}
+            sourceType={sourceType}
+            sourceFilter={router.query.source}
+            url={router.asPath}
+            title={title}
+          />
+        </StyledContent>
+        {linkedResults.matches.length > 0 && (
+          <StyledMenu>
+            <StyledMenuList>
+              <StyledMenuTitle>
+                Les articles pouvant vous interesser :
+              </StyledMenuTitle>
+              <ul>
+                {linkedResults.matches
+                  .filter(link => link.title !== title)
+                  .slice(0, 3)
+                  .map(link => (
+                    <li key={link.title}>
+                      <Link
+                        href={`/${getRouteBySource(link.source)}/${link.slug}`}
```

Comment:
i guess you need to add a `as` prop too ? 

like https://github.com/SocialGouv/code-du-travail-numerique/blob/master/packages/code-du-travail-frontend/src/search/SearchResults/Results.js#L36-L49

Reply:
In this case, the href value should be `/${getRouteBySource(link.source)}/[slug]` 
the as value should be `/${getRouteBySource(link.source)}/${link.slug}`

Reply:
Done